apps/src/studio/Item.js

```diff
@@ -1,460 +1,460 @@
-var Collidable = require('./collidable');
-var constants = require('./constants');
+import Collidable from './collidable';
+import * as constants from './constants';
```

Comment:
That only works for importing the default member of a module. I could created a `constants` object in constants.js and add everything to it, but this kind of import statement seems easier.